src/theme/components.js

```diff
@@ -249,6 +249,12 @@ const components = {
           outline: '1px auto',
         },
       },
+      select: {
+        '&:focus': {
+          backgroundColor: Colors.Background300Dark,
```

Comment:
Maybe unique classes would work better here. What do you think? 

```
    select: {
        '&.select-light': {
          '&:focus': {
            backgroundColor: Colors.SecondaryDefault,
          },
          '&:hover': {
            padding: 0,
            backgroundColor: Colors.SecondaryHover,
          },
        },
        '&.select-dark': {
          '&:focus': {
            backgroundColor: Colors.Background300Dark,
          },
          '&:hover': {
            padding: 0,
            backgroundColor: Colors.SecondaryHoverDark,
          },
        },
      },
```

But for this to work, you'd have to update both the dropdown components that use `Select`, so it's more work but at least it will move us away from `typestyle`.

https://github.com/Netacci/perfcompare/blob/6aa592eac274eef1fb470b86144aace635560fb3/src/components/Search/SearchDropdown.tsx#L61C10-L61C12

You'd have to add an `inputProps `with the new class to all the `Select` components for example:

```
inputProps={{
   classes: {
     select: mode === 'light' ? 'select-light' : 'select-dark',
      },
}}

```

Reply:
There's this flickering that happens when I hover over the select. Removing the `padding:0` seems to fix the issue

https://github.com/user-attachments/assets/05da47f5-1932-448d-a8d7-dd6d1fe46428

Reply:
Ahh, you're right! Okay, thanks for catching.

Reply:
Yes, that's because we have separate hover styles for the `selected:focus`. However, on regular hover, there's extra padding. 
<img width="795" alt="Screenshot 2024-12-17 at 15 34 06" src="https://github.com/user-attachments/assets/4ca85956-3ad5-44c8-b512-cf564511efe1" />

Could you set the hover padding to : padding: '3px 0px 5px' ? And see if that makes it look perfect

Reply:
There's only a flick when you hover on the sides, but that's ok for now. What's important that it stays one color on focus. Some other style finishes I'll take care of when I revamp the styles later.

Reply:
Alright. will do